### ISSUE TITLE
Small device filters modal is stuck in the "middle" of the page

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#506]
## Description

The small device filters modal appears in the middle of the "page" rather than the middle of the viewport. This means that if there are any results displayed it will be scrolled way down to the middle of the results view rather than in the center of the page.

## Reproduction

1. Change your browser to a small device width
2. Go to the search results page: https://wordpress.org/openverse/search/image?q=galah&license=cc0,pdm
3. Open the filters modal
4. You'll see a darkened background but no filter modal. Scroll down until you see the modal, it'll be about halfway down the results view.

## Expectation

The filter modal should appear front and center without having to scroll to find it.

To fix this we could try to use the new VModal component, but it'll require some modifications to match the current style. Alternatively we could just lift the styles and positioning logic from the VModal component and revisit this later to actually use the component here. That's probably the best "stop-gap" solution for this.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by sarayourfriend** [comment]
Blocked by https://github.com/WordPress/openverse-frontend/pull/447

There's no way to reliably vertically position the modal at the moment because of the WP header outside the iframe. Currently the modal container uses `height: 100%` but in the iframe that "100%" refers to the entire iframe height. We could use `100vh` but because the WP header takes up space outside the iframe, we'd actually need `calc(100vh - wp-header-height)` and I'm not sure how to do that.

If we merge the issue linked above first then we should be able to just use `100vh`.

**Comment by sarayourfriend** [comment]
Duplicate of https://github.com/WordPress/openverse-frontend/issues/536
